client/python/rappor.py

```diff
@@ -158,9 +158,11 @@ def get_rappor_masks(user_id, word, params, rand_funcs):
 
   assigned_cohort = rand_funcs.cohort_rand_fn(0, params.num_cohorts - 1)
   # Uniform bits for (*)
-  f_bits = rand_funcs.uniform_gen()
+  f_bits = rand_funcs.f_gen()
```

Comment:
Thanks for catching this bug.  I swapped them back and updated the variable names.